Jenkinsfile

```diff
@@ -6,17 +6,7 @@ def parseSemanticVersion(String version) {
 
 node('heavy && linux && docker') {
   checkout scm
-  def tag = sh(script: "git tag --contains", returnStdout: true).trim()
-  def isBump = !!tag
-  def isOnReleaseBranch = env.BRANCH_NAME == 'master'
-
-  if (!isOnReleaseBranch) {
-    return
-  }
-
-  if (!isBump) {
-    return
-  }
+  def releaseCommit
```

Comment:
can't the sh git command be run here after checkout?

Reply:
honestly I don't know.
I think we need to run it in a stage.
I kinda duplicated what worked in the cli repo until a build went through to be honest 😅